### ISSUE TITLE
As an account executive I'd like to use ListBuilder to configure and manage tasks for an operation

### ISSUE THREAD

**Issue by j0nscalet** [j0nscalet/QualityKitchen.API#19]
**Problem**
Account executives should be able to use ListBuilder to manage tasks for an operation.

**Solution**
- [x] ListBuilder needs to validate the 'Employee Roster'

- [x] ListBuilder needs to be connected to the API

- [ ] Google App Scripts should be included in source control
   - Should we create a separate repo for ListBuilder?
       - Can we have multiple READMEs in a repo?
       - It kind of is the tool that you use to manipulate the API
   - Find ListBuilder README - locate JavaScrip object pattern

**Comment by j0nscalet** [comment]
Documentation links for Google App Scripts, specifically Sheets:
Overview - https://developers.google.com/apps-script/guides/sheets 
Container Bound Scripts - https://developers.google.com/apps-script/guides/bound
Spreadsheet App API - https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet-app

**Comment by j0nscalet** [comment]
@mgpaja8 let's chat tomorrow to see where you're at with this. I'd like to have this "done done" by May 15th.

**Comment by j0nscalet** [comment]
@mgpaja8 All that's left to do is your comments and to build out the API bug #1 like we discussed. When you're done with the comments and cleanup let me know and I'll review it.

No need to go crazy with comments either, follow the format that I used (above methods and classes) and make comments in blocks where necessary. For example, a place where you're doing something a bit funky and other developers should know about it. 

Great work on this my man 😎
